src/components/augmentedReality/AugmentedRealityView.js

```diff
@@ -31,6 +31,7 @@ export const AugmentedRealityView = ({ sceneNavigator }) => {
   useEffect(() => {
     ViroARTrackingTargets.createTargets({
       [TARGET]: {
+        orientation: 'Up',
```

Comment:
sure we need this? 'Up' seems to be the default: https://github.com/ViroCommunity/viro/blob/7fa83641c807f3065291d0d3a29bfef4cacabaa9/ios/ViroReact/AR/Modules/VRTARTrackingTargetsModule.mm#L284

Reply:
if this is the correct file, i cannot find a default for android: https://github.com/ViroCommunity/viro/blob/7fa83641c807f3065291d0d3a29bfef4cacabaa9/android/viro_bridge/src/main/java/com/viromedia/bridge/module/ARTrackingTargetsModule.java

Reply:
When I go to the augmented reality page on Android, it says that the `orientation` prop is missing and crash the application.

Reply:
I think it is assigned as default data only for iOS. We can write this on discord.

Reply:
yes please share it with the community, it is something weird having a crash in this situation.